gradle.properties

```diff
@@ -8,4 +8,4 @@ serviceName=openlmis-notification
 # Current version of the service
 # Only a Release Manager should update this
 ######################################################################
-serviceVersion=4.1.1-SNAPSHOT
+serviceVersion=4.1.2-SNAPSHOT
```

Comment:
I will update version in other places (contract-tests, ref-distro, deployment) after this PR is merged.